source/agora/flash/Network.d

```diff
@@ -0,0 +1,331 @@
+/*******************************************************************************
+
+    Contains in-memory representation of Lightning Network topology
+
+    Copyright:
+        Copyright (c) 2020 BOS Platform Foundation Korea
+        All rights reserved.
+
+    License:
+        MIT License. See LICENSE for details.
+
+*******************************************************************************/
+
+module agora.flash.Network;
+
+import agora.flash.Config;
+import agora.flash.Channel;
+import agora.flash.Route;
+
+import agora.common.crypto.ECC;
+import agora.common.Hash;
+import agora.common.Set;
+import agora.common.Amount;
+
+private struct NetworkNode
+{
+    /// Channels that this NetworkNode is a part of
+    private Set!Hash[Point] channels;
```

Comment:
More specifically the key is a counter-party, and we can have multiple channels with another counter-party so that's why this is a set. Was that the intention?

Reply:
Yes, exactly.